src/graphics/program-lib/chunks/particle_blendAdd.frag

```diff
@@ -1,2 +1,3 @@
+    d_fog_factor=0.0; //for additive blend, this should be 0.0
```

Comment:
Right, but the comment doesn't really tell me why. 😄

Reply:
removed the comment - since there were no fog comments before